linode/firewall/framework_resource_test.go

```diff
@@ -126,7 +127,7 @@ func TestAccLinodeFirewall_minimum(t *testing.T) {
 					resource.TestCheckResourceAttr(testFirewallResName, "disabled", "false"),
 					resource.TestCheckResourceAttr(testFirewallResName, "inbound.#", "1"),
 					resource.TestCheckResourceAttr(testFirewallResName, "inbound.0.protocol", "TCP"),
-					resource.TestCheckResourceAttr(testFirewallResName, "inbound.0.ports", ""),
+					// resource.TestCheckResourceAttr(testFirewallResName, "inbound.0.ports", ""),
```

Comment:
probably uncomment this check?

Reply:
An empty string of `ports` is no longer a valid value in a recent API change. So it would become a `null` now in Terraform state.